sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryStatusListener.scala

```diff
@@ -38,8 +38,8 @@ import org.apache.spark.util.kvstore.KVIndex
  * UI data for both active and inactive query.
  */
 private[sql] class StreamingQueryStatusListener(
-    @transient val conf: SparkConf,
-    @transient val store: ElementTrackingStore) extends StreamingQueryListener {
+    val conf: SparkConf,
```

Comment:
This reverts the change in previous [PR](https://github.com/apache/spark/pull/41752/files#diff-175f4cb2acc5c47237fb39131a022a5557be21027cd6bbdf5433d0ce722d4d42)